### PR TITLE
update changelog for 6.1.0

### PR DESCRIPTION
**Pull request by gfuller1**
Change log for 6.1.0 release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
+## Version 6.1.0 (2020-09-29)
+* Support for Java 15
+* [Ability to add attributes to segments](https://github.com/newrelic/newrelic-java-agent/pull/67)
+* `Newrelic.addCustomParameter()` API [now supports boolean values](https://github.com/newrelic/newrelic-java-agent/pull/70)
+* Installation of [Real Time Java Profiler](https://docs.newrelic.com/docs/agents/java-agent/features/real-time-java-profiling-using-jfr-metrics)
+(which uses Java Flight Recorder data) via a daemon is now configurable in the agent. To enable in your configuration file, set:
+  ```
+    jmx:
+      linkingMetadataMBean: true
```
Comment (jasonjkeller) [inline CHANGELOG.md:18]: Is this something we're publicizing?

**Review by jasonjkeller — APPROVED** [review]

**Comment by jasonjkeller** [comment]
Gotta sign that CLA yo

File: CHANGELOG.md (hunk 1 above)
Comment (breedx-nr) [reply CHANGELOG.md:18]: Yeah, I don't think we should talk about the real time profiler here.  This is related to 1 of the 3 use cases for JFR.  I also think the current phrasing is confusing and possibly misleading.  Somebody could think that just setting that somehow enables JFR, which isn't true.  I say we should simplify this item.

Comment (gfuller1) [reply CHANGELOG.md:18]: @abhihub

Comment (abhihub) [reply CHANGELOG.md:18]: Good point. Lets not publicize this setting till Profiling is released. We can update the release notes at that point with a better description.

**Comment by gfuller1** [comment]
I signed the CLA

**Review by jasonjkeller — APPROVED** [review]
LGTM

**Pull request merged**